src/routine.cpp

```diff
@@ -24,6 +24,7 @@ namespace clblast {
 // The constructor does all heavy work, errors are returned as exceptions
 Routine::Routine(Queue &queue, EventPointer event, const std::string &name,
                  const std::vector<std::string> &routines, const Precision precision,
+                 std::initializer_list<const char *> source_pct,
```

Comment:
pad/copy/transpose — this contains the code for transformation routines (that can be used with custom main kernel).